### PR TITLE
fix(client): add toggleable warning for when codeally is down + remove 'new' and 'beta' from superblock titles

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by naomi-lgbt — COMMENTED** [review]
Would this require a re-deployment every time to toggle the codeAllyDown flag?

**Comment by moT01** [comment]
Yes, but the terminology is somewhat neutral in that it sort of says, "it may or may not be working", so we can leave it up if we want. CodeAlly has been down for a number of days - so we just want to get something up to let campers know. It's a temporary fix until we get [feature flags](https://github.com/freeCodeCamp/freeCodeCamp/issues/50171) working. Any thoughts or other ideas @naomi-lgbt?

**Review by ShaunSHamilton — APPROVED** [review]
It works, so I am still approving.

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -996,7 +996,8 @@
     "legacy-go-back": "Go to the current version of the curriculum.",
     "new-rwd-desc": "We have updated our Responsive Web Design Curriculum. If you were previously working on the RWD curriculum, your progress is still saved! You can find it under the Legacy Responsive Web Design section.",
     "new-rwd-article": "We encourage you to read about the changes we made and consider exploring the updated curriculum.",
```
[the remark this answers is not in the document]
Comment (moT01) [reply client/i18n/locales/english/intro.json:998]: Probably

File: config/misc.ts [hunk 2]
```diff
@@ -3,3 +3,4 @@ export const MAX_MOBILE_WIDTH = 767;
 export const TOOL_PANEL_HEIGHT = 37;
 export const SEARCH_EXPOSED_WIDTH = 980;
 export const DONATE_NAV_EXPOSED_WIDTH = 600;
+export const codeAllyDown = true;
```
[the remark this answers is not in the document]
Comment (moT01) [reply config/misc.ts:6]: No reason. I had it defined in the two places it's used and wanted a central spot for it - so I copy/pasted it here - didn't notice the casing. Maybe not the best place, but it works.

File: client/src/templates/Challenges/codeally/show.tsx [hunk 3]
```diff
@@ -250,6 +253,28 @@ class ShowCodeAlly extends Component<ShowCodeAllyProps> {
...
+            {codeAllyDown && superBlock === SuperBlocks.RelationalDb && (
+              <Row>
+                <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
+                  <Spacer size='medium' />
+                  <Alert bsStyle='danger'>
+                    <p>
+                      <Trans i18nKey='intro:misc-text.course-maintenance'>
+                        <a
+                          href='https://www.freecodecamp.org/news/how-to-run-freecodecamps-relational-databases-curriculum-using-docker-vscode-and-coderoad'
+                          rel='noreferrer'
```
[the remark this answers is not in the document]
Comment (moT01) [reply client/src/templates/Challenges/codeally/show.tsx:265]: You're probably right.

**Comment by moT01** [comment]
Unless there's a significant issue with this - I vote to go with it like this for now so we can try and get it deployed @ShaunSHamilton. I can fix those things up in another PR.

**Comment by moT01** [comment]
Let's put this on hold for now - looks like the courses are working again, so I don't think we need this right now. We should try and get that feature flags feature working.

**Review by ShaunSHamilton — APPROVED** [review]
👍

**Review by naomi-lgbt — APPROVED** [review]
Okie dokie

**Pull request merged**